rust/src/filecontainer.rs

```diff
@@ -25,6 +25,39 @@ extern {
     pub fn FileFlowToFlags(flow: *const Flow, flags: u8) -> u16;
 }
 
+// Unifying the definition of files SMBFiles and NFSFiles (HTTP2 too, soon)
```

Comment:
this comment should be updated to what it is now, it should not be about what it was or how we got there. That type of info should go into the git commit messages.